.github/workflows/wordpress-plugin-asset-update.yml

```diff
@@ -8,10 +8,11 @@ jobs:
     name: Push to master
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
+    - uses: php-actions/composer@v1
```

Comment:
Any reason to not use latest v5 of the composer action?
(same for the deployment action)

Reply:
No, I've updated the workflows to use the latest version of the actions.